### PR TITLE
feat:  Sort Currency list on priority basis (#698)

### LINKED ISSUE #698: Sort Currency list
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore
    
### Bug Report    

**Actual Behaviour**

Currency list is in no particular order

**Expected Behaviour**

Currency list should be sorted

**Would you like to work on the issue?**

Yes

Parent issue #689

### PR DESCRIPTION
**Pull request by AkshayCHD**
Fixes #698 

Changes: Added few important currencies at the top and sorted rest alphabetically in currency list.

### CODE REVIEW

**Comment by AkshayCHD** [comment]
@iamareebjamal Do I have to implement something like this.

**Comment by iamareebjamal** [comment]
Yes, the approach is correct. However, don't remove the already existing codes and also add currency of euro and GB.

**Comment by AkshayCHD** [comment]
@iamareebjamal I have added UK for GB, euro is already there for germany, and prevented removal of already existing codes. please review

**Comment by AkshayCHD** [comment]
@iamareebjamal I have added UK for GB, euro is already there for germany, and prevented removal of already existing codes. please review

File: app/src/main/java/org/fossasia/openevent/app/common/utils/core/CurrencyUtils.java [hunk 1]
```diff
@@ -70,6 +71,17 @@ private CurrencyUtils() {
...
+
+        Collections.sort(currencyCodes);
+
+        currencyCodes.add(0, Currency.getInstance(Locale.CANADA).toString());
+        currencyCodes.add(0, Currency.getInstance(Locale.CHINA).toString());
+        currencyCodes.add(0, Currency.getInstance(Locale.KOREA).toString());
+        currencyCodes.add(0, Currency.getInstance(Locale.GERMANY).toString());
+        currencyCodes.add(0, Currency.getInstance(Locale.JAPAN).toString());
+        currencyCodes.add(0, Currency.getInstance(Locale.UK).toString());
+        currencyCodes.add(0, Currency.getInstance(Locale.US).toString());
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/common/utils/core/CurrencyUtils.java]: This is inefficient. It shifts the whole list each time

**Comment by AkshayCHD** [comment]
@iamareebjamal please review

**Pull request merged**